azurerm/internal/services/applicationinsights/resource_arm_application_insights.go

```diff
@@ -89,6 +89,19 @@ func resourceArmApplicationInsights() *schema.Resource {
 
 			"tags": tags.Schema(),
 
+			"daily_cap": {
+				Type:         schema.TypeFloat,
+				Optional:     true,
+				Default:      100,
+				ValidateFunc: validation.FloatBetween(0, 1000),
+			},
+
+			"stop_send_notification_when_hit_cap": {
```

Comment:
This property is awfully odd, could we invert and reword it to
```suggestion
			"daily_traffic_cap_notifications_enabled": {
```

where true -> sends false, false -> send true

Reply:
Hi @katbyte,
agree with you awfully odd. But i think we should name it the other way. Because in general people will only touch this option if they want to stop notifications if cap is hit e.g. disabled = true.

**daily_data_cap_notifications_disabled**
true -> don't send messages
false -> send messages
default is false

Reply:
I'm ok with `daily_data_cap_notifications_disabled`, thanks!